pkg/apps/blackduck/components/rc/authentication/v1/component.go

```diff
@@ -25,25 +23,26 @@ type BdReplicationController struct {
 }
 
 func init() {
-	store.Register(blackduck.BlackDuckAuthenticationRCV1, NewBdReplicationController)
+	store.Register(types.BlackDuckAuthenticationRCV1, NewBdReplicationController)
 }
 
 // GetRc returns the RC
 func (c *BdReplicationController) GetRc() (*components.ReplicationController, error) {
-	containerConfig, ok := c.Containers[blackduck.AuthenticationContainerName]
+	containerConfig, ok := c.Containers[types.AuthenticationContainerName]
 	if !ok {
-		return nil, fmt.Errorf("couldn't find container %s", blackduck.AuthenticationContainerName)
+		return nil, fmt.Errorf("couldn't find container %s", types.AuthenticationContainerName)
 	}
 
-	if containerConfig.MaxMem == nil {
-		return nil, fmt.Errorf("Maxmem must be set for %s", blackduck.AuthenticationContainerName)
+	hubMaxMemory := 512
+	if containerConfig.MaxMem != nil {
```

Comment:
This should be != nil and > 512 or something right?  Otherwise you could end up with a negative number.

Is it really hubMaxMemory?  It's the java heap size right?  Any sane differentiation between that and what the hub is using?  If not, probably need a comment to explain why we are doing this 512 thing.